x-pack/plugins/osquery/public/scheduled_queries/scheduled_queries_table.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { find, uniq, map } from 'lodash/fp';
+import { EuiInMemoryTable, EuiBasicTableColumn, EuiLink } from '@elastic/eui';
+import React, { useCallback, useMemo } from 'react';
+import { UseQueryResult, useQueries } from 'react-query';
+
+import {
+  GetOneAgentPolicyResponse,
+  PackagePolicy,
+  agentPolicyRouteService,
+} from '../../../fleet/common';
+import { pagePathGetters } from '../../../fleet/public';
+import { useKibana, useRouterNavigate } from '../common/lib/kibana';
+import { useScheduledQueries } from './use_scheduled_queries';
+import { ActiveStateSwitch } from './active_state_switch';
+
+const ScheduledQueryNameComponent = ({ id, name }: { id: string; name: string }) => (
+  <EuiLink {...useRouterNavigate(`scheduled_queries/${id}`)}>{name}</EuiLink>
+);
+
+const ScheduledQueryName = React.memo(ScheduledQueryNameComponent);
+
+const renderName = (_: unknown, item: PackagePolicy) => (
+  <ScheduledQueryName id={item.id} name={item.name} />
+);
+
+const ScheduledQueriesTableComponent = () => {
+  const {
+    application: { getUrlForApp },
+    http,
+  } = useKibana().services;
+
+  const { data } = useScheduledQueries();
+
+  const uniqAgentPolicyIds = useMemo<string[]>(() => {
+    if (!data?.items) {
+      return [];
+    }
+
+    return uniq(map('policy_id', data.items));
+  }, [data?.items]);
+
+  const agentPolicies = useQueries(
```

Comment:
is there a way to do this in bulk? i was trying to use the list endpoint to get at the names, but from what i can tell it is omitted from that one.